src/shared-components/container/index.js

```diff
@@ -0,0 +1,26 @@
+import {
+  Container,
+  ClickableContainer,
+  MessageContainer,
+  Section,
+  Divider,
+  Image,
+} from './style';
+
+Container.Section = Section;
```

Comment:
no reason to have this as 3 separate things. clickable vs message should now be passed as a prop to outer container  i think

Reply:
fixed in 6e41aff